### PR TITLE
GitLab can use OIDC

### PR DESCRIPTION
**Pull request by kalexand-rh**
From https://trello.com/c/HruwzLtI/923-document-move-gitlab-to-oidc

@enj, PTAL

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@enj, did you backport this change, or is it valid in only 3.11?

**Comment by enj** [comment]
> @enj, did you backport this change, or is it valid in only 3.11?

3.11 only

File: install_config/configuring_authentication.adoc [hunk 1]
```diff
@@ -1543,9 +1543,12 @@ xref:../admin_guide/manage_rbac.adoc#admin-guide-manage-rbac[control user roles]
 === GitLab
 
 Set `GitLabIdentityProvider` in the `identityProviders` stanza to use
-link:https://gitlab.com/[GitLab.com] or any other GitLab instance as an identity provider, using the
+link:https://gitlab.com/[GitLab.com] or any other GitLab instance as an identity
+provider. If you use GitLab version 7.7.0 to 11.0, you connect using the
 link:http://doc.gitlab.com/ce/integration/oauth_provider.html[OAuth integration].
-The OAuth provider feature requires GitLab version 7.7.0 or higher.
+If you use GitLab version 11.1 or later, you can use link:
```
Comment (enj) [inline install_config/configuring_authentication.adoc]: `link` out of place?

**Comment by enj** [comment]
Just one minor comment, otherwise LGTM.

**Comment by kalexand-rh** [comment]
@enj, I've addressed your comment. Thanks!

@stuartchuan, will you PTAL?

@openshift/team-documentation PTAL

**Comment by ahardin-rh** [comment]
LGTM!

**Pull request merged**